### DISCUSSION TITLE
[Solved] show more information in Board Layout

### DISCUSSION THREAD

**Discussion by Qwyntex** [community/community#6799] [Projects and Issues] [answered]
**(I did not see that my own fields were disabled in the settings)**

Instead of just seeing the name, you should be able to see more information on the card. Maybe in a dropdown menu or directly on the card right to the name. 

Showing more information like a deadline directly on the card would allow you to organize everything a bit more. You could also then maybe sort the cards in their column, maybe even for each column separately. In that way you could sort one column for the importance of the issues and the next one for the Deadline. 
That would make everything a bit more organized.

**Comment by timoyates** [comment, marked as answer]
Hi @Lupusthe2nd, thanks for the feedback! Have you tried going to your board view's settings menu and toggling the visibility of your fields there? Custom fields like `deadline` as you mention don't appear by default, but you can turn them on (along with labels, assignees, etc.)

**Reply by Qwyntex** [reply]
no i did not see that thank you. is there any way to sort by them?

**Reply by timoyates** [reply]
Not yet, but we're tracking that feedback too. Cheers!

**Reply by Qwyntex** [reply]
Ah ok
Also that is probably not really related to this thread but can you show multiple views at once?
in any way have good day

**Reply by timoyates** [reply]
@Lupusthe2nd That's not possible at the moment but thanks for the question! Do you mind sharing more about the scenario? We're curious why one view isn't cutting it for you (e.g. not enough filter types etc)

**Reply by Qwyntex** [reply]
Two views would be really helpful if you would want to compare two tasks. Also it would be nice to use the column view and the chart view simultaneously because the column view is clear and easy to understand whereas the chart view is better at displaying data. (Maybe if the selected task could light up in the views to make it easier for the user).

TL;Dr
It would make it faster to use
